src/components/NewLocationPage/ShareButton/ShareButton.style.tsx

```diff
@@ -0,0 +1,6 @@
+import styled from 'styled-components';
+import { OutlinedButton } from 'components/ButtonSystem';
+
+export const StyledShareButton = styled(OutlinedButton)`
+  font-weight: normal;
```

Comment:
lets keep the default font weight for now. so i think you can nix this style file and use OutlinedButton directly in ShareButton.tsx